Monika After Story/game/script-topics.rpy

```diff
@@ -6937,6 +6937,306 @@ label monika_citizenship:
+        m 2eksdlc "You're {i}always{/i} so supportive..."
+        show monika 5lkc at t11 zorder MAS_MONIKA_Z with dissolve
+        m 5lkc "..."
+        m "Something must have happened, I know you love me too much to have done this on purpose."
+        m 5euc "Yeah..."
+        m 2eka "It's okay, [player], I understand sometimes things happen that can't be avoided."
+        m 2esa "Whenever things calm down, I'll give my speech to you again."
+        m 2eua "I still really want to share it with you..."
+        m "So please, let me know when you have time to hear it, okay?"
+    
+    else:
+        $ mas_loseAffection(reason="you didn't listen to my speech. I worked really hard on it.")
+
+        m 2ekc "..."
+        m 6ektdc "[player]! You weren't even paying attention!"
+        m 6lktdc "You have no idea how much that hurts, especially after how much of work I put into it..."
+        m 6ektdc "I just wanted to make you proud of me..."
+        m 6dstsc "..."
+
+    return
+
+label monika_grad_speech_ignored_lock:
+    #Second timeout, lock speech
+    hide screen mas_background_timed_jump
+    #Set false for modified dialogue in the random
+    $ persistent._mas_pm_monika_listened = False
+    $ hideEventLabel("monika_grad_speech_call",lock=True,depool=True)
+    
+    if mas_isMoniAff(higher=True):
+        $mas_loseAffection(modifier=10)
+        m 6dstsc "..."
+        m 6ektsc "[player]? {w=0.5}You...{w=0.5} You weren't...{w=0.5} listening...{w=0.5} Again?{w=1}{nw}"
+        m 6dstsc "I...{w=0.5} I thought last time it was unavoidable...{w=0.5} but...{w=0.5} twice?{w=1}{nw}"
+        m 6ektsc "You knew how much...{w=0.5} how much this meant to me...{w=1}{nw}"
+        m "Am I really that...{w=0.5} that boring to you?{w=1}{nw}"
+        m 6lktdc "Please...{w=1} don't ask me to recite it again...{w=1}{nw}"
+        m 6ektdc "You obviously don't care."
+
+    else:
+        $ mas_loseAffection(modifier=5)
+        m 2efc "..."
+        m 2wfw "[player]! I can't believe you did this to me again!{w=1}{nw}"
+        m 2tfd "You knew how upset I was the last time and you still couldn't be bothered to give me four minutes of your attention?{w=1}{nw}"
+        m "I don't ask that much of you...{w=1}{nw}"
+        m 2tfc "I really don't.{w=1}{nw}"
+        m 2lfc "All I ever ask is that you care... That's it.{w=1}{nw}"
+        m 2lfd "And yet you can't even {i}pretend{/i} to care about something you {i}know{/i} is so important to me.{w=1}{nw}"
+        m 2dkd "...{w=1}{nw}"
+        m 6lktdc "You know what, nevermind. Just...{w=0.5} nevermind.{w=1}{nw}"
+        m 6ektdc "I won't bother you about this anymore."
+
+    return
+
+label monika_grad_speech:  
+
+    # clear selected track
+    $ songs.current_track = songs.FP_NO_SONG
+    $ songs.selected_track = songs.FP_NO_SONG
+    #play some grad music
+    play music "mod_assets/sounds/amb/PaC.ogg" fadein 1.0
+    $ store.songs.enabled = False
```

Comment:
rather set this var to False, use the function `mas_MUMURaiseShield`, which is designed to block music menu both visually and interactively